.input.swagger/azure/2024-05-01-preview/inference.yaml

```diff
@@ -0,0 +1,7526 @@
+openapi: 3.0.0
```

Comment:
I wonder if we should ingest directly from azure-rest-api-specs like the submodule addition for openai-openapi; it feels a bit funny to pull one but copy the other, but then I'm not sure what the overhead starts looking like with many submodules, either -- I have to imagine there's a reason that the language repositories just use tool-driven sparse checkout the way they do.

I love having it side by side like this, though, as it'd be really easy to set up a semi-automatic diff (pull the commit hashes, do a temp string replacement, diff)

Reply:
I would be more than happy to actually pull a 2nd submodule. I have no idea where their swagger files are published, if there is a public repo available more than happy to setup a sparse checkout submodule of the versions as we add support to them. (I have no idea how that is done, but hopefully is not too much of a pain 😄 )